ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -116,9 +121,17 @@ export const ErrorComponent = (props: any) => {
   ) : null
 }
 export function validateListOfPoints(coordinates: any[], mode: string) {
-  let message = ''
+  let message = null
   const isLine = mode.includes('line')
   const numPoints = isLine ? 2 : 4
```

Comment:
I know you didn't introduce this, but I think this should be changed from 4 to 3, since a triangle is a valid polygon.

Reply:
The 4 is because a polygon requires the first and last points to be the same for it to be closed. The number of unique points is the length of the coordinates array minus one.